prelude/toolchains/msvc/vswhere.py

```diff
@@ -202,6 +202,76 @@ def write_tool_json(out, tool):
     out.write(j)
 
 
+# for use with the ewdk to grab the environment strings
+def get_env(ewdkdir: str):
+    """
+    Inspiration taken from the following:
+    http://pythonwise.blogspot.fr/2010/04/sourcing-shell-script.html (Miki Tebeka)
+    http://stackoverflow.com/questions/3503719/#comment28061110_3505826 (ahal)
+    """
+
+    # We need to write the script that will make the important variables available
+    # This gets run from the temporary build directory, so this does not pollute the sources
+    vcVarsExtract = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                "VcVarsExtract.bat")
+    if not os.path.exists(vcVarsExtract):
+        with open(vcVarsExtract, "w") as f:
+            f.write("@echo off\r\ncall %%1 %%2 %%3 %%4 %%5 %%6 %%7 %%8 %%9 > NUL\r\nset")
+
+    batch_file = os.path.join(ewdkdir, "BuildEnv", "SetupBuildEnv.cmd")
+
+    # This does not work as a list of strings, unfortunately
+    cmd = vcVarsExtract + ' "' + batch_file + '" ' + "amd64"
+
+    output = subprocess.check_output(cmd)
+    output = output.decode("utf-8")
```

Comment:
I don't get why list of arguments doesn't work. I think this should work.
```suggestion
    output = subprocess.check_output([tmp.name, env_script, "amd64"]).decode("utf-8")
```